gnocchi/utils.py

```diff
@@ -103,22 +102,26 @@ def get_coordinator_and_start(url):
 def to_timestamps(values):
     try:
         values = list(values)
-        if isinstance(values[0], numbers.Real):
-            times = pd.to_datetime(values, utc=True, box=False, unit='s')
-        elif (isinstance(values[0], datetime.datetime) or
-              is_valid_timestamp(values[0])):
-            times = pd.to_datetime(values, utc=True, box=False)
+        if isinstance(values[0], (numpy.datetime64, datetime.datetime)):
+            times = numpy.array(values)
         else:
             try:
                 float(values[0])
             except ValueError:
-                times = (utcnow() + pd.to_timedelta(values)).values
+                if is_valid_timestamp(values[0]):
```

Comment:
can you add a note to explain what each of these conditions is trying to handle? it seems non-trivial aside from line105